src/components/MyReservation.astro

```diff
@@ -0,0 +1,48 @@
+---
+
+---
+
+<div class="w-full">
```

Comment:
Måske lægge de to felter ovenpå hinanden i <640px, så tror jeg den klarer alle skærmstørrelser